src/scripts/main.js

```diff
@@ -5,4 +5,31 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const dashboard = document.querySelector('.dashboard');
+
+people.forEach(person => {
+  const row = document.createElement('tr');
+
+  row.insertAdjacentHTML('beforeend', `
+       <td>
+         ${person.name}
+         </td>
+         <td>
+         ${person.sex}
+         </td>
```

Comment:
```suggestion
  row.insertAdjacentHTML('beforeend', `
       <td>
         ${person.name}
       </td>
       <td>
         ${person.sex}
       </td>
```
what about formatting your code for better readability?))